docs/scenario_editor/roads.md

```diff
@@ -0,0 +1,51 @@
+## Creating the Road Network
+
+### Roads
+The monoDrive Scenario Editor provides a tool for creating road networks that vehicles in the simulation
+can drive on. 
+
+To add a road to a map, use the __Actor Place Mode__ in the editor, and search for "monoDrive Road". Then simply drag and drop it into the scene.
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/monoDrive_road_tool.PNG"/>
+
+Roads are defined by a _reference line_ that represents the basic geometry of the road. Lanes can be added to a road in both the forward and reverse directions, and the lanes follow the path of the reference line. 
+
+A forward lane begins at the location where the road is placed, and ends at the last spline point in the reference line. Similarly, reverse lanes begin at the last spline point and end at the location where the road was placed in the scene.
+
+In the figure below, the blue line is the road's reference line. The blue dots are spline points along the road's reference line.
+
+<img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/lane_tool/WikiPhotos/scenario_editor/roads/reference_line.PNG"/>
```

Comment:
Maybe make this image wider to see the blue line. It's hard to see. class='wide_img'